utilities_common/db.py

```diff
@@ -0,0 +1,6 @@
+from swsssdk import ConfigDBConnector
+
+class Db(object):
```

Comment:
Should we call this `ConfigDb`? Maybe we will add other DB connectors here in the future?